From 18b41fcb663a11fcd8ccc27bd951cc150456291b Mon Sep 17 00:00:00 2001
From: Maximilian Bosch <maximilian@mbosch.me>
Date: Fri, 16 Jan 2026 21:50:11 +0100
Subject: [PATCH] admin: file dir doesn't have to be a mountpoint

---
 authentik/admin/files/backends/file.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/authentik/admin/files/backends/file.py b/authentik/admin/files/backends/file.py
index 6d60d5f236..720223231f 100644
--- a/authentik/admin/files/backends/file.py
+++ b/authentik/admin/files/backends/file.py
@@ -52,7 +52,6 @@ class FileBackend(ManageableBackend):
         # file creation on fresh installs.
         return (
             self._base_dir.exists()
-            and (self._base_dir.is_mount() or (self._base_dir / self.usage.value).is_mount())
             or (settings.DEBUG or settings.TEST)
         )
 
-- 
2.51.2

